transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionTransformers.java

```diff
@@ -44,6 +48,13 @@ public String getSubsystemName() {
     @Override
     public void registerTransformers(SubsystemTransformerRegistration subsystemRegistration) {
         ChainedTransformationDescriptionBuilder chainedBuilder = TransformationDescriptionBuilder.Factory.createChainedSubystemInstance(CURRENT_MODEL_VERSION);
+        // 5.0.0 --> 4.0.0
+        ResourceTransformationDescriptionBuilder builderEap71 = chainedBuilder.createBuilder(CURRENT_MODEL_VERSION, MODEL_VERSION_EAP71);
+        builderEap71.getAttributeBuilder()
+                .setDiscard(new DiscardAttributeChecker.DiscardAttributeValueChecker(MAXIMUM_TIMEOUT.getDefaultValue()), MAXIMUM_TIMEOUT)
+                .addRejectCheck(new RejectAttributeChecker.SimpleAcceptAttributeChecker(MAXIMUM_TIMEOUT.getDefaultValue()), MAXIMUM_TIMEOUT)
```

Comment:
AFAICT this should be RejectAttributeChecker.DEFINED. The discard on the line above deals with the 'allowed' value. I've tried and the tests pass.